layout/css/gdp-figure.css

```diff
@@ -11,14 +11,33 @@ a{
   text-decoration:none;
 }
 
-#gdp-fig-two h1 h2 h3 p text,
```

Comment:
@lindsaycarr so for future CSS adventures, CSS doesn't let you list out all the elements that you want styles like this.  CSS is looking for a text in a p tag that is in a h3 tag that is in a h2 tag that is an h1 tag.  That is the reason you weren't seeing the changes you wanted.  LESS lets you do simplification type things with CSS, just gotta get it into Vizlab some how! 😄

Reply:
:pray: :hourglass_flowing_sand: